cardano-db-sync-extended/app/cardano-db-sync-extended.hs

```diff
@@ -29,7 +30,14 @@ main = do
   cmd <- Opt.execParser opts
   case cmd of
     CmdVersion -> runVersionCommand
-    CmdRun params -> runDbSyncNode extendedDbSyncNodePlugin params
+    CmdRun params -> do
+        let configFile = enpConfigFile params
+        enc <- readSyncNodeConfig configFile
+        let prometheusPort = dncPrometheusPort enc
```

Comment:
We do not need all these `let` bindings?
```
prometheusPort <- dncPrometheusPort <$> readSyncNodeConfig (enpConfigFile params)
```
Removing the let bindings shows that `configFile` and `enc` are un-needed and the only thing we actually need is `prometheusPort`.

Reply:
I prefer a style where 3 lines of code are written down if that means that it will be clearer what the intention is. I can do what you asked as well.